docs/installation/migrate.rst

```diff
@@ -47,43 +47,64 @@ To migrate |sr| to |ccloud|, follow these steps:
     .. code:: bash
 
         curl -X PUT -H "Content-Type: application/json" -H "Authorization: Basic dGVuYW50Mi1rZXk6bm9oYXNo" "http://destregistry:8081/mode" --data '{"mode": "IMPORT"}'
```

Comment:
We have a precedent in the CCloud quickstart to use this command instead to pass in creds:

```
curl -u <schema-registry-api-key>:<schema-registry-api-secret> ......
```

Can we keep that consistent here as well?